packages/cli-repl/src/cli-repl.ts

```diff
@@ -39,9 +42,14 @@ class CliRepl {
    */
   async connect(driverUri: string, driverOptions: NodeOptions): Promise<void> {
     console.log(i18n.__(CONNECTING), driverUri);
+    // @ts-ignore
```

Comment:
What was the problem here? 'ts-ignore' is dangerous, there are many workarounds that are safer, maybe i can help you to find one.

Reply:
maybe this didn't work: https://github.com/choojs/nanobus/blob/master/index.d.ts, :/ .. or it wants to be used as class like: `new Nanobus('mongosh')`.

Reply:
Another a neat way is to use an interface in place of the actual type:

```
interface Bus {
  emit(string, ...any) => void; 
}
```

An then just

```
  private bus: Bus
```

That would work in typescript since classes does not have to implement interfaces explicitly. This is also a sort of pattern.

Reply:
Forgot to mention the quickest fix of all: `private bus: any`

Reply:
Probably the cleanest way is just doing `new Nanobus('mongosh')`. I'll do that!